privacyidea/lib/token.py

```diff
@@ -2084,7 +2084,8 @@ def check_token_list(tokenobject_list, passw, user=None, options=None):
         message_list = ["matching {0:d} tokens".format(len(valid_token_list))]
         # write serial numbers or something to audit log
         for token_obj in valid_token_list:
-            token_obj.inc_count_auth_success()
+            if not is_true(get_from_config(key="no_auth_counter")):
```

Comment:
As we call ``not is_true(...)`` three times, what do you think about putting it to a separate function like ``is_auth_counter_enabled()`` (or so?)

Reply:
since ``is_true`` only occurs in one function, I set a local variable for that.